Clarify CI Job Descriptions (#38132)
DataDog/documentation: content/en/data_observability/cicd.md

##### General settings

| Setting                            | Description                                                          |
| ---------------------------------- | -------------------------------------------------------------------- |
| `Run on Draft Pull/Merge Requests` | Enable this option to run the check on draft pull or merge requests. |

#### Drift Detection

> Drift detection compares the branch state of your data to a baseline and flags any deviations. For Datadog to understand which models ran as part of a CI pipeline, you _must_ send Open Lineage events from your CI job; Datadog uses these events as triggers for Drift Detection checks. See the [OpenLineage setup documentation][6] for instructions on how to set up Open Lineage.

Reviewer: ```suggestion
Drift detection compares the current state of your data on the branch to a baseline and flags any deviations. For Datadog to understand which models ran as part of a CI pipeline, you must send OpenLineage events from your CI job. Datadog uses these events as triggers for drift detection checks. See the [OpenLineage setup documentation][6] for instructions on how to set up OpenLineage.
```

| `Run on Draft Pull/Merge Requests` | Enable this option to run the check on draft pull or merge requests.                                                                                          |
| `Threshold`                        | The threshold for drift detection (for example, `0.1` for 10% drift). If a metric exceeds this threshold, it shows up as a warning in the check results.      |
| `Downstream Checks`                | When a dbt model changes, drift detection checks are generated for it and any downstream dbt models. This setting controls how far downstream the checks run. |

##### dbt Cloud

| Setting      | Description                                                                                                                                                                                                                   |
| ------------ | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |

> | `CI Job URL` | The locator for your dbt Cloud CI job: this is the job that's triggered by pull requests, materializes dbt models for CI, and sends Open Lineage events to Datadog. These typically look like `https://cloud.getdbt.com/...`. |

Reviewer: ```suggestion
| `CI Job URL` | The locator for the dbt Cloud CI job that's triggered by pull requests, materializes dbt models for CI, and sends OpenLineage events to Datadog. These typically look like `https://cloud.getdbt.com/...`. |
```

##### dbt Core

| Setting            | Description                                                                                                                                                                                                                                        |
| ------------------ | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |

> | `CI Job Name`      | The name of the job that's triggered by pull requests, materializes dbt models for CI, and sends Open Lineage events to Datadog.                                                                                                                   |

Reviewer: ```suggestion
| `CI Job Name`      | The name of the job that's triggered by pull requests, materializes dbt models for CI, and sends OpenLineage events to Datadog.                                                                                                                   |
```

> | `CI Job Namespace` | The `OPENLINEAGE_NAMESPACE` variable specified when sending Open Lineage from the specified above job. See [Set the environment variables][7]. If you don't set this variable when sending Open Lineage events, you don't need to specify it here. |

Reviewer: ```suggestion
| `CI Job Namespace` | The OPENLINEAGE_NAMESPACE variable specified when sending OpenLineage events from the job specified above. See [Set the environment variables][7]. If you don't set this variable when sending OpenLineage events, you don't need to specify it here. |
```
